src/python_tools/robot_broadcast_sender.py

```diff
@@ -0,0 +1,41 @@
+from software.proto.announcement_pb2 import Announcement
+import socket
+import fcntl
+import struct
+import time
+
+
+def get_ip_address(ifname):
```

Comment:
would be good to write some docs here and explain how your getting the broadcast address

Reply:
function comments would be nice too

Reply:
added